src/components/toast/toast.style.js

```diff
@@ -38,7 +38,10 @@ const ToastStyle = styled(MessageStyle)`
     position: relative;
     margin-right: ${isCenter ? "auto" : "30px"};
   `}
- 
+  
```

Comment:
I'm not sure we want a outline here, can we check with UX to confirm?

Reply:
Will do.

Reply:
for what it's worth we don't currently have an outline on Dialog etc. when their containers are autofocused after opening - we should try for consistency unless UX say otherwise